packages/connect-web-bench/package.json

```diff
@@ -9,8 +9,8 @@
     "@bufbuild/connect-web": "0.2.1",
     "@bufbuild/protobuf": "0.1.1",
     "@bufbuild/protoc-gen-es": "0.1.1",
-    "esbuild": "^0.15.6",
-    "google-protobuf": "^3.20.1",
-    "grpc-web": "^1.3.1"
+    "esbuild": "^0.15.10",
+    "google-protobuf": "^3.21.0",
+    "grpc-web": "^1.4.1"
```

Comment:
These runtime dependencies should match the code generator version. 

Can you update packages/connect-web-bench/buf.gen.yaml? 
You will also need to run `make bench` to update to benchmark results.

Reply:
Ah, will do.

Reply:
I'll also update the notion doc to remind anyone doing dependabot updates to do this.

Reply:
Alright, updated.